docs/recommended-workflow.md

```diff
@@ -19,37 +19,35 @@ It's expected that you already do this each work morning for the previous work d
 
 ### Write progress report
 
-#### details
+#### Details
 
 * Due 5pm each Friday
 * To: rcummings@nerevu.com
 * Subject: Progress Update for week ending `MM-DD-YY`
 
-#### contents
+#### Contents
 
-```md
-# Accomplishments
+**Accomplishments**
 
 * Completed `foo` task
 * Learned `bar` skill
 * Updated documentation on `baz`
 
-# Struggles / Issues / Concerns
+**Struggles / Issues / Concerns**
 
 * Couldn't figure out how to do `xyz`
 * It takes too long to do `abc` on program `qrs`
 
-# Future Plans
+**Future Plans**
 
 * Complete `foo` and `bar` tasks
 * Learn `baz` skill
 * Updated documentation on `xyz`
 
-# Questions / Requests
+**Questions / Requests**
 
 * What is the proper way to `abc`?
 
-# Recommendations
+**Recommendations**
 
 * Process `abc` is inefficient because of `xyz`. We should improve it by implementing `foo`.
-```
```

Comment:
I wanted this to stay formatted as md inside the code blocks

Reply:
Oh I see what you mean now! Okay I added back the markdown formatting in recommended-workflow and pushed the changes to the Updates branch. It should be ready for the pull request to go through. Let me know if my edits did now go through.